datashape/tests/test_parser.py

```diff
@@ -611,9 +624,11 @@ def test_funcproto_no_return_type(sym):
         parse('(int64, int32) ->', sym)
 
 
-def test_empty_tuple_fails(sym):
-    with pytest.raises(DataShapeSyntaxError):
-        parse('()', sym)
+def test_empty_tuple(sym):
```

Comment:
Can you add a test for a function signature like this `() -> datetime`? Doesn't have to be `datetime` could be something else, just need to test a function signature with no arguments that returns a value